rockcraft/utils.py

```diff
@@ -77,11 +77,7 @@ def confirm_with_user(prompt, default=False) -> bool:
 
     choices = " [Y/n]: " if default else " [y/N]: "
 
-    reply = str(input(prompt + choices)).lower().strip()
-    if reply and reply[0] == "y":
-        return True
+    reply = input(prompt + choices).lower().strip()
+    possible_answers = {" y ": True, " n ": False}
 
-    if reply and reply[0] == "n":
-        return False
-
-    return default
+    return possible_answers.get(reply[0].lower().strip(), default) if reply else default
```

Comment:
why .lower here and on line 80? same for strip

Reply:
Lower : Because of the commentary of the function "True if answer starts with [yY], False if answer starts with [nN],
        otherwise the default." so i lower to get only have the key y or n in my dict
Strip : i will change it it's because of black i need to add space in the get